lib/services/android-project-service.ts

```diff
@@ -152,11 +153,19 @@ class AndroidProjectService extends projectServiceBaseLib.PlatformProjectService
 	}
 
 	public canUpdatePlatform(currentVersion: string, newVersion: string): IFuture<boolean> {
-		return Future.fromResult<boolean>(true);
+		return Future.fromResult(true);
 	}
 
-	public updatePlatform(currentVersion: string, newVersion: string): IFuture<void> {
-		return Future.fromResult();
+	public updatePlatform(currentVersion: string, newVersion: string, canUpdate: boolean): IFuture<boolean> {
+		return (() => {
+			if(semver.eq(newVersion, AndroidProjectService.MIN_RUNTIME_VERSION_WITH_GRADLE)) {
+				this.$commandsService.tryExecuteCommand(process.argv[2], ["remove"].concat(process.argv.slice(4))).wait();
```

Comment:
Imho, this is a highly suspicious code - our project service depends on the UI code, which means that a lower-layered code depends on the higher level one. Can we approach this in another way?